spec/sierra_bib_spec.rb

```diff
@@ -148,4 +148,12 @@
       expect(vf5[0]['extracted_content'][0]).to eq('agriculture and education, planting the seeds of opportunity')
     end
   end
-end
\ No newline at end of file
+
+  describe 'find_oclcnum' do
+    it 'gets oclcnum from MARC::Record' do
+      b = SierraBib.new('b52446219')
+      expect(b.oclcnum).to eq(b.marc.oclcnum)

```

Comment:
As things are with this PR, I think b.oclcnum and b.marc.oclcnum will always equal nil because there was never a b.find_oclcnum to set @oclcnum for the bib and marc. We could "expect(b.find_oclcnum)..." or call b.find_oclcnum after initializing b, but other comments on the PR may obviate the need for that anyway.

Reply:
Yep the other changes you suggested obviated doing anything more about this.